resources/assets/js/components/Datatable.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="container">
+    <div class="row">
+      <div class="col-12">
+        <table id="DataTable" class="table table-striped table-bordered" cellspacing="0" width="100%">
+        </table>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+  
+
+  export default {
+    props: ['columns', 'dataUrl'],
+    data() {
+      return {
+        tableData: {},
+        columnsDatatables: []
+      }
+    },
+    mounted() {
+      console.log(this.dataUrl);
+      console.log(this.columns);
+
+      this.columnsDatatables = this.columns.map(function(column) {
+        return {'title': column, 'data': column};
+      });
+
+      axios.get(this.dataUrl)
+        .then(response => {
+          this.tableData = response.data;
+
+          $('#DataTable').DataTable({
```

Comment:
Can't actually do that because `DataTable()` needs to be called on a JQuery object